### ISSUE TITLE
[Question] How to limit bandwidth for export block usage of nfs-ganesha v4.x+ ?

### ISSUE THREAD

**Issue by duynguyen33** [nfs-ganesha/nfs-ganesha#1101]
Hi team,

As for the question above, I don't know if there is any way the nfs-ganesha service usage protocol NFS ver 4.x+ can limit network bandwidth.

**Comment by dang** [comment]
Correct, we don't currently have a way to throttle bandwidth.  You can do that at the host level for the entire ganesha, but not per-export.

**Comment by xiaods** [comment]
> throttle bandwidth

use tc to limit bandwidth on Linux

**Comment by ffilz** [comment]
We are working on some quality of service enhancements, including per-export bandwidth budget.

I'll leave this open to represent work is in progress on that.

**Comment by deeraj-patil** [comment]
@duynguyen33 : 
Code for QOS is available in current upstream branch.
please refer src/doc/man/ganesha-qos-config.rst for QOS config.

@ffilz : I think we can close this issue.

**Comment by ffilz** [comment]
Closing as resolved.
